app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,39 @@
+module SessionsHelper
+  def log_in(user)
+    session[:user_id] = user.id
+  end
+
+  def logged_in?
+    !current_user.nil?
+  end
+
+  def log_off
+    forget(current_user)
+    session.delete(:user_id)
```

Comment:
Forget about using `session`

Reply:
I can do that, and the login would work, but without a session, I can not define when the user is in and when he can log out, that is the issue for me